lib/sources/subquery-indexer/index.ts

```diff
@@ -115,23 +115,6 @@ export default class SubqueryIndexerSource implements SourceInterface {
           offset += this.paginationLimit;
         }
 
-        if (!graphQlResponse) {
```

Comment:
Isn't it possible that graphQlResponse is null or undefined?

Reply:
actually you're right. but it case of error we should not go to the next page but just throw an error

Reply:
Sounds legit